### PR TITLE
Add new entities

### PR DESCRIPTION
**Pull request by Ludovic33Fr**
Add to the list of entities these two third party : 

- Hubvisor
- Truscommander

### CODE REVIEW

File: data/entities.js [hunk 1]
```diff
@@ -12048,4 +12048,19 @@ module.exports = [
...
+    categories: ['consent-provider'],
+    domains: ['*.trustcommander.net'],
+    examples: ['cdn.trustcommander.net', 'privacy.trustcommander.net'],
+  },
+  {
+    name: 'Hubvisor',
+    homepage: 'https://www.hubvisor.io',
+    categories: ['ad'],
+    domains: ['*.hubvisor.io'],
+    examples: ['stream.hubvisor.io', 'stream.hubvisor.io'],
```
Comment (patrickhulce) [inline data/entities.js:12064]: were these meant to be different examples?

**Review by patrickhulce — APPROVED** [review]
thanks for the contribution @Ludovic33Fr!

LGTM % duplicate example :)

File: data/entities.js (hunk 1 above)
Comment (Ludovic33Fr) [reply data/entities.js:12064]: sorry it's a mistaken i would write :  ['cdn.hubvisor.io', 'stream.hubvisor.io']

i can do another PR

Comment (patrickhulce) [reply data/entities.js:12064]: no need for a new PR :) thanks for the clarification
```suggestion
    examples: ['cdn.hubvisor.io', 'stream.hubvisor.io'],
```

**Pull request merged**